pages/_app.js

```diff
@@ -0,0 +1,7 @@
+import '../pods/covid-spain-chart/base.css'
+import '../pods/covid-spain-chart/map.css'
+
+// This default export is required in a new `pages/_app.js` file.
+export default function MyApp({ Component, pageProps }) {
```

Comment:
Why this file is _app.js? Is it something temporary, if it's so please add a comment

Should it be app.tsx?

Reply:
Because it is the way how next.js manage the css files:
https://nextjs.org/docs/basic-features/built-in-css-support

Reply:
Fixed to CSS modules, check slack for more details